### PR TITLE
Asterisk Application support

### PR DESCRIPTION
**Pull request by TheGreatDoc**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

PR 183 from librenms-agent contains the script to make this work

https://github.com/librenms/librenms-agent/pull/183

### CODE REVIEW

**Comment by jozefrebjak** [comment]
I can not get that script https://github.com/librenms/librenms-agent/blob/master/snmp//asterisk ... i cant see it. Where can I find it. I want to test this PR

**Comment by TheGreatDoc** [comment]
@jozefrebjak While its merged, you can take it from my forked repo https://github.com/TheGreatDoc/librenms-agent/blob/asterisk/snmp/asterisk

File: doc/Extensions/Applications.md [hunk 1]
```diff
@@ -115,6 +116,27 @@ snmpwalk <various options depending on your setup> localhost NET-SNMP-EXTEND-MIB
 (If you get error like "Can't locate LWP/Simple.pm". libwww-perl needs to be installed: apt-get install libwww-perl)
 2. On the device page in Librenms, edit your host and check the `Apache` under the Applications tab.
 
+### Asterisk
+A small shell script that reports various Asterisk call status.
+
+##### SNMP Extend
+1. Copy the [asterisk script](https://github.com/librenms/librenms-agent/blob/master/snmp//asterisk) to `/etc/snmp/` on your asterisk server.
```
Comment (Cormoran96) [inline doc/Extensions/Applications.md]: `snmp//asterisk`
There is a double character /

Comment (TheGreatDoc) [reply doc/Extensions/Applications.md]: Fixed!

**Pull request merged**